docs/integrations/keras_integration.rst

```diff
@@ -0,0 +1,597 @@
+
+Neptune-Keras Integration
+*************************
+
+What will you get with this integration
+=======================================
+
+In this tutorial, you'll learn how to integrate Neptune into your Keras application, and at the end you'll be able to efficiently track and monitor model training, metrics, images and even datasets.
+
+Video [1min screencast]
+
+|Keras| is a high level interactive API that can be used to prototype, build and train neural networks. \
+Keras acts as an interface for the TensorFlow library, and in the latest version of Tensorflow,\
+Keras was officially integrated under the name |tf.keras|. The original Keras is still available,\
+but in this tutorial, you'll be using the Tensorflow version \
+of Keras (TF Keras). Don't worry, the API is totally the same. For more explanation \
+on TF Keras, check out this |link|. At the end of this tutorial you will learn: \
+
+* How to integrate Neptune with Keras to effectively track metrics and model training. 
+* How to easily reproduce experiments and aid collaboration in your project. 
+
+**Note:** This integration works with library==tf.keras v0.24.1 and neptune-contrib v0.24.1 
+
+Links:
+|source code| |open in colab|
+
+.. |Keras| raw:: html
+
+    <a href="https://keras.io" target="_blank">Keras</a> 
+
+.. |tf.keras| raw:: html
+
+    <a href="https://www.tensorflow.org/api_docs/python/tf/keras" target="_blank">tf.keras</a>
+
+.. |link| raw:: html
+
+    <a href="https://www.pyimagesearch.com/2019/10/21/keras-vs-tf-keras-whats-the-difference-in-tensorflow-2-0" target="_blank">link</a>
+
+.. |open in colab| raw:: html
+
+    <a href="https://colab.research.google.com/drive/1ITLOePfJh9rNR1jis5gBES2InDo_8Iz4?usp=sharing" target="_blank">open in colab</a>
+
+
+Quickstart
+==========
+
+
+Introduction
+------------
+Neptune is a lightweight experiment management tool that helps you keep track of your machine learning experiments. With Neptune, you can: 
+
+* `Monitor ML runs live <https://docs.neptune.ai/getting-started/quick-starts/how-to-monitor-live.html#use-cases-monitor-runs-live>`_
+
+* `Organize ML experimentation <https://docs.neptune.ai/getting-started/quick-starts/how-to-organize-experiments.html#use-cases-organize-ml-experiments>`_
+
+* `Compare and debug ML experiments and models <https://docs.neptune.ai/getting-started/quick-starts/how-to-compare-experiments.html#use-cases-compare-and-debug-experiments>`_
+
+* `Share results of experiments with your team <https://docs.neptune.ai/getting-started/quick-starts/how-to-share-results.html#use-cases-share-results-with-team>`_
+
+* `Clean up your Jupyter workflow without git <https://docs.neptune.ai/getting-started/quick-starts/how-to-clean-up-jupyter.html#use-cases-clean-jupyter-workflow>`_
+
+In this tutorial, you'll build a simple spam classifier using Keras. The task is a binary classification problem,\
```

Comment:
Remember this is not a blog post but rather a docs page. 

Showing an example like that is probably a bit too much (especially if they need to download some new dataset and stuff.